src/cosmospy/_typing.py

```diff
@@ -1,14 +1,15 @@
 import sys
```

Comment:
Why was the `typing` module made private? Can we revert that change?

Reply:
I took this as it was before without really thinking about it. I will change it